input/labor-anonymized.md

```diff
@@ -33,9 +34,9 @@
 * A theoretical connection between Qi and Arrows in category theory was drawn.
 * The idea of a Qi-themed event was suggested
 * The Qi design challenge was organized
-* An illustrative example to motivate design improvements to the `feedback` form of the language was provided
-* The design of `feedback` was improved
-* The PR improving the design of `feedback` was reviewed
```

Comment:
I think in this particular case, the original ones here reflected a later stage of discussion. I can revert it after merging though.

Reply:
Ah, you mean it should have been changes in the evaluation trees rather than the inputs?

Reply:
No, just that the versions that were replaced here were what we had originally changed the text _to_. I restored them in [this commit](https://github.com/countvajhula/dia-qi/commit/98b2c49b9b27e33c9f21511c306e7611c92e3b99). Lmk if that doesn't look right to you.